src/Grid/Grid.levers.js

```diff
@@ -1,14 +1,110 @@
 /**
  * Available HTML elements of the grid.
  */
-export const gridElements = ['div', 'section', 'article', 'main', 'footer', 'aside', 'header', 'nav', 'form'];
+export const gridElements = [
+  'div',
+  'section',
+  'article',
+  'main',
+  'footer',
+  'aside',
+  'header',
+  'nav',
+  'form'
+];
 
 /**
  * Grid gap or not
  */
 export const gridGap = [true, false];
 
 /**
- * Available number of grid columns for all breakpoints (1 to 12)
+ * Width of the flex cell (n-of-12 columns).
  */
-export const gridNumCols = Array.from({length: 12}, (_, i) => i + 1);
+
+export const gridCols = {
+  'xs': {
+    '1': 'su-grid-cols-1',
+    '2': 'su-grid-cols-2',
+    '3': 'su-grid-cols-3',
+    '4': 'su-grid-cols-4',
+    '5': 'su-grid-cols-5',
+    '6': 'su-grid-cols-6',
+    '7': 'su-grid-cols-7',
+    '8': 'su-grid-cols-8',
```

Comment:
At first I was a little worried that these might not work if an actual number is entered as a prop, e.g., xs={3}, but it seems to work for both number and string. That's great!